README.md

```diff
@@ -59,6 +58,23 @@ In Haml, use `=`, not `-`:
 
 For more on Haml syntax, see the "Haml" section below.
 
+The `code` helper supports [Rouge](https://github.com/jayferd/rouge) instance formatter options such as:
+
+* `line_numbers`
+* `start_line`
+* `css_class`
+* `wrap`
+
+To use these formatter options per code block, include them in a hash as the second argument. e.g.
+
+```erb
+<% code("ruby", {:line_numbers => true, :start_line => 7}) do %>
```

Comment:
The surrounding curly brackets aren't required, I'd probably remove them for succinctness.